RadarSDK/Include/Radar.h

```diff
@@ -765,6 +774,33 @@ typedef void (^_Nonnull RadarRouteMatrixCompletionHandler)(RadarStatus status, R
                        units:(RadarRouteUnits)units
            completionHandler:(RadarRouteMatrixCompletionHandler)completionHandler NS_SWIFT_NAME(getMatrix(origins:destinations:mode:units:completionHandler:));
 
+/**
+ Sends a custom event.
+
+ @param name The name of the event.
+ @param metadata The metadata associated with the event.
+ @param completionHandler A completion handler.
+
+ @see https://radar.com/documentation/api#send-event
+ */
++ (void)sendEvent:(NSString *)name
+     withMetadata:(NSDictionary *_Nullable)metadata
+completionHandler:(RadarSendEventCompletionHandler)completionHandler NS_SWIFT_NAME(sendEvent(name:metadata:completionHandler:));
+
+/**
+ Sends a custom event with a manually provided location.
+
+ @param name The name of the event.
+ @param location The location of the event.
```

Comment:
Add `@param metadata The metadata associated with the event.`

Reply:
good catch